docs/snippets.py

```diff
@@ -935,5 +939,25 @@ def test_list_rows_as_dataframe(client):
     assert len(df) == table.num_rows  # verify the number of rows
 
 
+@pytest.mark.skipif(geopandas is None, reason="Requires `geopandas`")
+def test_query_results_as_geodataframe(client):
```

Comment:
OK, will do.

Reply:
Done.  I ended up moving it to samples/geography.

The management of the requirements.txt there is a little confusing as it only uses released versions of `google-cloud-bigquery`. Hopefully, it will work out in CI. :)

I ended up adding lots of requirements.